### PR TITLE
Use Assert Class for assertions and fix other reviewed changes

### PR DESCRIPTION
**Pull request by satyasekharcvb**
### What does this PR do?
Use Assert Class for assertions and fix other reviewed changes

## The PR fulfils these requirements:

[ X ] Tests for the proposed changes have been added/updated.
[ X ] Code linting and formatting were performed.

### CODE REVIEW

File: force-app/main/default/classes/TestAccountController.cls [hunk 1]
```diff
@@ -18,7 +18,7 @@ public class TestAccountController {
         List<Account> accts = AccountController.getAccountList();
         Test.stopTest();
 
-        System.assertEquals(accts.size(), 10);
+        Assert.areEqual(accts.size(), 10);
```
Comment (pozil) [inline force-app/main/default/classes/TestAccountController.cls:21]: @satyasekharcvb can you fix the order of all asserts parameters: it's expected then, actual.
```suggestion
        Assert.areEqual(10, accts.size());
```

File: force-app/main/default/classes/TestLMSVisualforceController.cls [hunk 2]
```diff
@@ -51,7 +51,7 @@ private class TestLMSVisualforceController {
         insert testContact;
 
         // Verify that test data is all ready to go
-        System.assertNotEquals(
+        Assert.areNotEqual(
```
Comment (pozil) [inline force-app/main/default/classes/TestLMSVisualforceController.cls:54]: Switch to `Assert.isNull`

**Review by pozil — CHANGES_REQUESTED** [review]
@satyasekharcvb can you do a quick fix for me before I merge?

File: force-app/main/default/classes/TestAccountController.cls (hunk 1 above)
Comment (satyasekharcvb) [reply force-app/main/default/classes/TestAccountController.cls]: Thank you for pointing this @pozil! I have fixed it wherever necessary.

**Review by pozil — APPROVED** [review]
Thanks @satyasekharcvb, good to go.

**Pull request merged**